src/renderer/components/ft-video-player/ft-video-player.css

```diff
@@ -4,4 +4,23 @@
 
 .ftVideoPlayer {
   width:100%;
+}
+
+:deep(.sponsorBlockMarker), :deep(.chapterMarker) {
+  position: absolute;
+  opacity: 0.6;
+}
+
+:deep(.sponsorBlockMarker) {
+  height: 100%;
+  background-color: var(--primary-color);
+}
+
+:deep(.chapterMarker) {
+  height: 120%;
+  top: -10%;
+  width: 1%;
+  border-radius: 999px; /* make sure they are rounded properly*/
+  z-index: 2;
+  background-color: var(--accent-color);
 }
```

Comment:
Why still no final newline?
We already set it in `.editorconfig`
https://github.com/FreeTubeApp/FreeTube/blob/development/.editorconfig#L9

Reply:
Whoops, fixed it.

Reply:
Your editor/IDE might need to be setup to support `.editorconfig`
https://editorconfig.org/#download